liquid/conf/views.py

```diff
@@ -6,14 +6,14 @@
 from django.forms import ValidationError
 from conf.models import Company
 from utils.django_mailman.models import List
-from utils.group_decorator import group_admin_required
+from utils.group_decorator import group_admin_required, check_group_admin
 import simplejson as json
 
 def main(request):
 	return render_to_response('conf/main.html',{"section":"conf", "page":"main"},context_instance=RequestContext(request))
 
 def jobfair(request):
-    if request.user.is_authenticated():
+    if request.user.is_authenticated() and check_group_admin(['Conference', '!Company'], request):
```

Comment:
> Won't they be redirected to login instead of getting the permission denied?

Yes - the intent is that they should see the "what this page is for + login" page rather than a nondescriptive "permission denied" one.

Maybe the best solution is to redirect them to a version of the login page without the login form?

Looping in @zmmille2 in case he has any comments.